.gitignore

```diff
@@ -2,9 +2,8 @@ build
 node_modules
 /rules
 /customRules
-customRules.js
-base.js
-react.js
-untyped.js
+/*.js
+/src/**/*.js
```

Comment:
See this https://github.com/Shopify/tslint-config-shopify/pull/63#discussion_r120491394 for background. 

However, using `ts-node` solves all this extra work. Going to revert this.